torch/distributed/tensor/parallel/api.py

```diff
@@ -82,6 +87,14 @@ def parallelize_module(  # type: ignore[return]
 
     torch._C._log_api_usage_once("torch.distributed.tensor.parallel.parallelize_module")
 
+    # instantiate a TP RNG state tracker if it's not there
+    if (
+        is_rng_supported_mesh(device_mesh) and
+        not isinstance(random._rng_tracker, TensorParallelRNGTracker)
+    ):
+        random._rng_tracker = TensorParallelRNGTracker()
```

Comment:
let's put that in another PR maybe, because I'm not quite sure how this argument should be used. Is it an optional RNG tracker argument, and we use the default value when it's None? But we don't want user to know RNG tracker, right? That's kinda unknown to us for now.